lib/ansible/modules/cloud/azure/azure_rm_availabilityset.py

```diff
@@ -93,9 +97,19 @@
 
 RETURN = '''
 state:
-    description: Current state of the availability set
+    description: Current state of the availability set.
     returned: always
     type: dict
+    example: {
```

Comment:
It should not only give an example. But some yaml files, you can refer the servicebus module’s doc